ratbagd/ratbagd-profile.c

```diff
@@ -507,8 +507,8 @@ ratbagd_profile_set_report_rate(sd_bus *bus,
 		return r;
 
 	/* basic sanity check */
-	if (rate > 5000)
-		rate = 5000;
+	if (rate > 8000)
+		rate = 8000;
```

Comment:
Which devices has a poll rate of 8000Hz? Even 5000Hz is over the top, iirc USB only supports 1000Hz anyway and the one device I've seen in the past that advertises 2000Hz does so by using multiple event nodes and then randomly sending the events through whichever one is next in the internal fw ring buffer.

Reply:
I think there is only one such mouse in the market right now - Raver Viper 8K. I don't know whether it uses a technique like the one described though.

Reply:
I second this particular change. You can do 8kHz in USB HS, some high-end devices are doing this. I and designing an open source/hardware mice with a friend that supports this.